app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java

```diff
@@ -576,8 +579,10 @@ public void onClick(View view) {
             }
 
             case R.id.customtab_close: {
-                erase();
```

Comment:
I filed #616 to clarify what we want to do here, since I can't find any documented decisions either way. It seemed weird to have different behaviour for (system) back button vs close button, but that's probably for UX to decide!